src/dbus_api.rs

```diff
@@ -317,9 +325,11 @@ fn create_dbus_filesystem<'a>(mut dbus_context: DbusContext) -> dbus::Path<'a> {
                               STRATIS_BASE_PATH,
                               dbus_context.get_next_id().to_string());
 
+    let interface_name = format!("{}.{}", STRATIS_BASE_SERVICE, "filesystem");
+
```

Comment:
this binding strictly isn't needed.. did you consider the alternative and decide to do it this way for clarity or line length reasons?